README.md

```diff
@@ -92,8 +92,8 @@ The `master` branch is the development version of Saleor and it may be unstable.
 
 The current stable version is 2.11 and you should use this version for all three components:
 
-- Saleor: https://github.com/mirumee/saleor/releases/tag/2.11.0
-- Dashboard: https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0
+- Saleor: https://github.com/mirumee/saleor/releases/tag/2.11.1
+- Dashboard: https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.1
 - Storefront: https://github.com/mirumee/saleor-storefront/releases/tag/2.11.0
```

Comment:
Shouldn't we update the storefront to 2.11.1 too? It could be misleading when Storefront will have a different version.

Reply:
Yep that's a good point but there were no changes in the storefront to release as 2.11.1. We assume that major and minor numbers must match for all the components to work together (so all `2.11.x` must be compatible).